.github/workflows/cross.yml

```diff
@@ -344,6 +345,7 @@ jobs:
           echo "GOROOT=${{ steps.setup_msys2.outputs.msys2-location }}\${{ matrix.sys }}\lib\go" >> $GITHUB_ENV
           echo "CC=${{ steps.setup_msys2.outputs.msys2-location }}\${{ matrix.sys }}\bin\${{ matrix.cc }}.exe" >> $GITHUB_ENV
           echo "CXX=${{ steps.setup_msys2.outputs.msys2-location }}\${{ matrix.sys }}\bin\${{ matrix.cxx }}.exe" >> $GITHUB_ENV
+          echo "CMAKE_GENERATOR=Ninja" >> $GITHUB_ENV
```

Comment:
The build environment is picking up the wrong "make" executable ([`C:/mingw64/bin/make.exe`](https://github.com/aws/aws-lc-rs/actions/runs/14980922168/job/42084583367#step:6:321)) which appears to be incompatible with the "msys2" environment. So we use Ninja instead.